pages/integrations/inputs/palo_alto_networks_input.rst

```diff
@@ -82,4 +85,26 @@ Version 7.1
 * `7.1 - Threat Log Fields <https://www.paloaltonetworks.com/documentation/71/pan-os/pan-os/monitoring/syslog-field-descriptions#67983>`_
 * `7.1 - System Log Fields <https://www.paloaltonetworks.com/documentation/71/pan-os/pan-os/monitoring/syslog-field-descriptions#74679>`_
 
+
+PAN-OS 9 Input
+--------------
+
+This input auto detect if the data that is ingested is from Version 9.0 or 9.1. Since the release of Graylog 3.3.6 the later is supported automatically and will work out of the box.
+
+The previous possible adjustments are not longer needed.
+
+We have included a links to a few recent versions here for reference.
+
+Version 9.1
+
+* `9.1 - Traffic Log Fields <https://docs.paloaltonetworks.com/pan-os/9-1/pan-os-admin/monitoring/use-syslog-for-monitoring/syslog-field-descriptions/traffic-log-fields.html>`_
+* `9.1 - Threat Log Fields <https://docs.paloaltonetworks.com/pan-os/9-1/pan-os-admin/monitoring/use-syslog-for-monitoring/syslog-field-descriptions/threat-log-fields.html>`_
+* `9.1 - System Log Fields <https://docs.paloaltonetworks.com/pan-os/9-1/pan-os-admin/monitoring/use-syslog-for-monitoring/syslog-field-descriptions/system-log-fields.html>`_
+
+Version 9.0
+
+* `9.0 - Traffic Log Fields <https://docs.paloaltonetworks.com/pan-os/9-0/pan-os-admin/monitoring/use-syslog-for-monitoring/syslog-field-descriptions/traffic-log-fields.html>`_
+* `9.0 - Threat Log Fields <https://docs.paloaltonetworks.com/pan-os/9-0/pan-os-admin/monitoring/use-syslog-for-monitoring/syslog-field-descriptions/threat-log-fields.html>`_
+* `9.0 - System Log Fields <https://docs.paloaltonetworks.com/pan-os/9-0/pan-os-admin/monitoring/use-syslog-for-monitoring/syslog-field-descriptions/system-log-fields.html>`_
```

Comment:
The PAN-OS 9.x Input supports the following log types: CONFIG, CORRELATION, HIP, GLOBAL_PROTECT (the only one that differs significantly between 9.1 and 9.1.3), SYSTEM, THREAT, and TRAFFIC

Reply:
changed the lists to in the PR